src/ui/Island/Realm.tsx

```diff
@@ -264,6 +280,26 @@ export default function Realm({
           {...pathProps}
         />
       ))}
+      {/* This path is used to create the blinking effect */}
+      {isNew && !isRealmDisplayed(id) && (
+        <>
+          {paths.map((path, index) => (
+            <animated.Path
+              key={index}
+              x={x}
+              y={y}
+              data={path}
+              width={width}
+              height={height}
+              listening={false}
+              onClick={handleRealmClick}
+              {...blinkingProps}
+            />
+          ))}
+          {/* This is the "New realm" label */}
```

Comment:
I don't think this comment is needed - component has a good name to describe what is going on here :P